addons/ai/functions/fnc_findEmptyStatic.sqf

```diff
@@ -7,11 +7,34 @@
         Finds empty statics / vehicle nearby
         
     Parameter(s):
-        0: Static
+        0: Unit hit <OBJECT>
+        1: Object causing the damage <OBJECT>
+        2: Damage dealt <SCALAR>
+        3: Unit causing the damage <SCALAR>
         
     Return Value:
         Nothing
 */
 
+#include "script_component.hpp"
+
+params ["_unit", "", "", ""];
+
+if (
+    !(alive _unit)
+) exitWith {};
+
+[{
```

Comment:
Not sure what this is doing here. [{...}] is loading some code into an array, which is never getting called.